AssetAdministrationShellRegistryServiceSpecification/V3.1_SSP-003.yaml

```diff
@@ -0,0 +1,1975 @@
+openapi: 3.1.0
+info:
+  title: >-
+    DotAAS Part 2 | HTTP/REST | Bulk Asset Administration Shell Registry Service
+    Specification (DRAFT, not official!)
+  description: >-
+    The Bulk Profile of the Asset Administration Shell Registry Service
+    Specification as part of the [Specification of the Asset Administration
+    Shell: Part 2](http://industrialdigitaltwin.org/en/content-hub).  
+
+    The server processes all requests asynchronously, ensuring atomic
+    operations. This means that requested manipulations will either fully
+    succeed or be completely rolled back if any error occurs during the
+    process.  
+
+    Publisher: Industrial Digital Twin Association (IDTA) April 2023
+  contact:
+    name: Industrial Digital Twin Association (IDTA)
+    email: info@idtwin.org
+  license:
+    name: CC BY 4.0
+    url: https://creativecommons.org/licenses/by/4.0/
+  version: V3.1.0_SSP-003
+  x-profile-identifier: >-
+    https://admin-shell.io/aas/API/3/1/AssetAdministrationShellRegistryServiceSpecification/SSP-003
+servers:
+- url: 'https://admin-shell.io/api/v3.1'
+- url: 'https://example.com/'
+
+paths:
+  /bulk/shell-descriptors:
+    post:
+      tags:
+        - Async Bulk Asset Administration Shell Registry API
+      summary: >-
+        Creates multiple new Asset Administration Shell Descriptors, i.e.
+        registers multiple Asset Administration Shells
+      operationId: BulkPostAssetAdministrationShellDescriptor
```

Comment:
In https://admin-shell-io.github.io/aas-specs-antora/IDTA-01002/v3.1/specification/interfaces.html#_asset_administration_shell_registry_interface it is 
- CreateBulkAssetAdministrationShellDescriptors
- PutBulk...
- DeleteBulk...

correct conformant to last naming decision